src/resources/items/ItemList.tsx

```diff
@@ -145,14 +145,14 @@ const getFilters = (resource?: string): React.ReactElement[] => {
   if (resource === constants.R_ALL_ITEMS) {
     filters.push(
       <BooleanFilter<Item>
```

Comment:
@TahaKhanAbdalli - we need the previous objects for these.

When something is added to a scheduled dispatch or destruction, the `destruction` and `dispatch` ids get set. But, we only know it has been `dispatched` or `destroyed` when the `...Date` has been set.

Was this change introduced to resolve some change?